src/Angor/Client/Pages/Investor.razor

```diff
@@ -22,11 +14,19 @@
 
 @inject IJSRuntime JS
 
+@using Angor.Client.Models
```

Comment:
the using should be over the inject why did it change here?

Reply:
You're correct, for some reason my reformat does that..